.github/workflows/build_push_service.yml

```diff
@@ -6,20 +6,169 @@ name: sc-app-build-push
 
 # Controls when the action will run.
 on:
+  push:
+    branches:
+      - develop
+      - master
+      # release branches are of e.g. release/25.3.0
+      - release/[0-9]+.[0-9]+.[0-9]+
+      # OPS-1664 naming convention <branch>/<JIRA-Ticket ID>-<Jira_Summary>
+      - hotfix/[A-Z]+-[0-9]+-[a-zA-Z0-9]+_[a-zA-Z0-9]+*
+      - feature/[A-Z]+-[0-9]+-[a-zA-Z0-9]+_[a-zA-Z0-9]+*
```

Comment:
Why are it on so many cases at once, it's created many special cases to handle.

Reply:
No, Workflow should just fail on old naming scheme.